package/boot/uboot-rockchip/patches/202-rockchip-rk3399-Add-support-for-FriendlyARM-NanoPi-R.patch

```diff
@@ -0,0 +1,308 @@
+From 7cc86e66ccf7723de76016e0a0d4b563d89f44b0 Mon Sep 17 00:00:00 2001
+From: Tianling Shen <cnsztl@gmail.com>
+Date: Fri, 8 Jan 2021 05:55:50 +0000
+Subject: [PATCH] rockchip: rk3399: Add support for FriendlyARM NanoPi R4S
+
+This adds support for the NanoPi R4S from FriendlyArm.
+
+Rockchip RK3399 SoC
+1GB DDR3 or 4GB LPDDR4 RAM
+Gigabit Ethernet (WAN)
+Gigabit Ethernet (PCIe) (LAN)
+USB 3.0 Host Port x 2
+MicroSD slot
+Reset button
+WAN - LAN - SYS LED
+
+[initial DTS, defconfig]
+Co-developed-by: Jensen Huang <jensenhuang@friendlyarm.com>
+Signed-off-by: Jensen Huang <jensenhuang@friendlyarm.com>
+Co-developed-by: Marty Jones <mj8263788@gmail.com>
+Signed-off-by: Marty Jones <mj8263788@gmail.com>
+Signed-off-by: Tianling Shen <cnsztl@gmail.com>
+---
+ arch/arm/dts/Makefile                      |   1 +
+ arch/arm/dts/rk3399-nanopi-r4s-u-boot.dtsi |   9 +
+ arch/arm/dts/rk3399-nanopi-r4s.dts         | 185 +++++++++++++++++++++
```

Comment:
Please send this upstream, https://www.denx.de/wiki/U-Boot/Patches

Reply:
@peterwillcn Fine, I'll try do it later, but please make it clear here first.

Reply:
Backporting patches/device support from upstream sources is preferred.

Reply:
> Backporting patches/device support from upstream sources is preferred.

Apologize, however I'm not mastering the usage of mailing list as well as getting known the rules of each project,
so I would like to add it here firstly if possible...

Reply:
Update: now I had have sent the patches to kernel, and see what feedback I would receive then...

Reply:
Thanks! For reference https://patchwork.kernel.org/project/linux-rockchip/list/?series=430187

Reply:
Sorry but I had forgotten to remove the hostname in led labes, and the change should be included in v2.